### ISSUE TITLE
Migrations files needs to inherit from ActiveRecord::Migration[RAILS_VERSION]

### ISSUE THREAD

**Issue by loicginoux** [solidusio-contrib/solidus_related_products#19]
When running `bundle exec rails g solidus_related_products:install`

I had the following error:
```
StandardError: Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreateRelationTypes < ActiveRecord::Migration[4.2]
```

I had to add the my current rails app version to all the generated migration files like this for example:

```
class CreateRelationTypes < ActiveRecord::Migration[5.1]
```

See https://github.com/solidusio/solidus/wiki/Testing-extensions-against-multiple-Solidus-versions#migrations

**Comment by prwhitehead** [comment]
The gem can no longer be installed with Solidus (2.4.2) because of this issue. I'm going to have to fork it.

**Comment by askl56** [comment]
@prwhitehead has this issue been resolved?

**Comment by prwhitehead** [comment]
Looks like I cloned the repo: https://github.com/builtbybuffalo/solidus_related_products/commit/46542b5c216d551e212d001d2962fc0344d8d80e

**Comment by aitbw** [comment]
Closed via 9027e60
